_posts/2020-09-09-#18267.md

```diff
@@ -10,14 +10,46 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
+This pull request implements the portion of
+[BIP325](https://github.com/bitcoin/bips/blob/master/bip-0325.mediawiki)
+responsible for connecting to a signet network and validating blocks (it does
+not implement the code required by miners to sign signet blocks). We are
+touching consensus-critical code today! Let's make sure we do it right :-)
+
 
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- kallewoof has an (outdated)
```

Comment:
```diff
- kallewoof has an (outdated)
+- The PR author, [kallewoof](https://github.com/kallewoof), has an (outdated)
```

Reply:
Maybe explain in what way the gist is outdated. I also made a review comment in the PR that details, step by step, how one can manually test the PR.

Reply:
Th. review comment with the testing steps is here: https://github.com/bitcoin/bitcoin/pull/18267#issuecomment-620569419

Reply:
done